### PR TITLE
Casmuser-3291 for 2.7.1

### PR DESCRIPTION
**Pull request by darrenn-jackson**
#### Summary and Scope

- Fixes CASMUSER-3291

##### Issue Type

- Docs Pull Request

Uses correct names for the final COS playbook and CFS layer

#### Prerequisites

- [x] I have included documentation in my PR (or it is not required)
- [] I tested this on internal system (x) (if yes, please include results or a description of the test)
 
#### Risks and Mitigations
 

Low risk: Content-only change. No change in docs structure or formatting.

### CODE REVIEW

**Review by keopp-hpe — APPROVED** [review]

**Comment by darrenn-jackson** [comment]
@keopp-hpe this PR originally targeted `release/uan-2.7` instead of `release/uan-2.7.1` because these changes are specifically for fixing UAN 2.7.1. After I re-targeted this PR for 2.7.1, I had to resolve a bunch of merge conflicts because my branch contained 2.7.3 specific changes. I think I fixed them all.

File: docs/portal/developer-portal/operations/Create_UAN_Boot_Images.md [hunk 1]
```diff
@@ -247,8 +246,8 @@ See also [*HPE Cray EX System Software Stack Installation and Upgrade Guide for
 
 ```yaml
 session_templates:
-- name: uan-2.4.0
-  image: cray-shasta-uan-sles15sp3.x86_64-2.3.25
+- name: uan-2.7.3
+  image: cray-shasta-uan-sles15sp5.x86_64-2.7.3
```
Comment (darrenn-jackson) [inline docs/portal/developer-portal/operations/Create_UAN_Boot_Images.md:250]: @keopp-hpe for 2.7.1, should the image line now say `image: cray-shasta-uan-sles15sp5.x86_64-2.7.1`

File: docs/portal/developer-portal/operations/Create_UAN_Boot_Images.md [hunk 2]
```diff
@@ -247,8 +246,8 @@ See also [*HPE Cray EX System Software Stack Installation and Upgrade Guide for
 
 ```yaml
 session_templates:
-- name: uan-2.4.0
-  image: cray-shasta-uan-sles15sp3.x86_64-2.3.25
+- name: uan-2.7.3
```
Comment (darrenn-jackson) [inline docs/portal/developer-portal/operations/Create_UAN_Boot_Images.md:249]:
```suggestion
- name: uan-2.7.1
```

File: docs/portal/developer-portal/install/Install_the_UAN_Product_Stream.md [hunk 3]
```diff
@@ -143,14 +149,14 @@ Customers must finish the installation or upgrade of the UAN product before boot
...
-UAN provides RPMs used on UAN nodes. The RPMs are uploaded to Nexus as part of the installation process.
+UAN provides third-party RPMs used on UAN nodes. The RPMs are uploaded to Nexus as part of the installation process.
 
 The following Nexus raw repositories are created:
 
 - uan-2.7.XX-sle-15sp4
 - uan-2.7.XX-sle-15sp3
 
-The following Nexus group repositories are created and reference the preceding Nexus raw repos.
+The following Nexus group repository is created and reference the preceding Nexus raw repos.
```
Comment (darrenn-jackson) [inline docs/portal/developer-portal/install/Install_the_UAN_Product_Stream.md:159]:
```suggestion
The following Nexus group repositories are created and reference the preceding Nexus raw repos.
```

**Review by darrenn-jackson — COMMENTED** [review]
one fix.

File: docs/portal/developer-portal/operations/Create_UAN_Boot_Images.md (hunk 1 above)
Comment (keopp-hpe) [reply docs/portal/developer-portal/operations/Create_UAN_Boot_Images.md]: Yes, for 2.7.1 it should reflect that.

File: docs/portal/developer-portal/operations/Create_UAN_Boot_Images.md [hunk 4]
```diff
@@ -247,8 +246,8 @@ See also [*HPE Cray EX System Software Stack Installation and Upgrade Guide for
 
 ```yaml
 session_templates:
-- name: uan-2.4.0
-  image: cray-shasta-uan-sles15sp3.x86_64-2.3.25
+- name: uan-2.7.1
+  image: cray-shasta-uan-sles15sp5.x86_64-2.7.3
```
Comment (darrenn-jackson) [inline docs/portal/developer-portal/operations/Create_UAN_Boot_Images.md:250]:
```suggestion
  image: cray-shasta-uan-sles15sp5.x86_64-2.7.1
```

File: docs/portal/developer-portal/operations/Create_UAN_Boot_Images.md (hunk 1 above)
Comment (darrenn-jackson) [reply docs/portal/developer-portal/operations/Create_UAN_Boot_Images.md]: Just fixed it in the latest commit.

**Review by keopp-hpe — APPROVED** [review]

**Pull request merged**